projectq/types/_qubit_test.py

```diff
@@ -60,21 +60,26 @@ def test_basic_qubit_comparison(id0, id1, expected):
     assert qubit2 != qubit0
     assert qubit2 != qubit1
     # Same engines
-    if expected:
-        assert qubit0 == qubit1
-    else:
-        assert not (qubit0 == qubit1)
+    assert (qubit0 == qubit1) == expected
 
 
 def test_basic_qubit_hash():
     fake_engine = "Fake"
-    qubit0 = _qubit.BasicQubit(fake_engine, 0)
-    qubit1 = _qubit.BasicQubit(fake_engine, 1)
-    assert hash(qubit0) != hash(qubit1)
-    qubit0.id = -1
-    qubit1.id = -1
-    # Important that weakref.WeakSet in projectq.cengines._main.py works.
```

Comment:
This is an important comment and still valid afterwards, hence deleting it is not allowed